lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -541,20 +543,18 @@ def __init__(self, precision=0.005, coord_name_to_integrate="height",
                 'negative' corresponds to the values within the array
                 decreasing as the array index increases.
         """
-        self.precision = precision
-        self.coord_name_to_integrate = coord_name_to_integrate
-        self.start_point = start_point
-        self.end_point = end_point
-        self.direction_of_integration = direction_of_integration
+        self.wet_bulb_temperature_plugin = (
+            WetBulbTemperature(precision=precision))
+        self.integration_plugin = Integration(
+            coord_name_to_integrate, start_point=start_point,
+            end_point=end_point,
+            direction_of_integration=direction_of_integration)
 
     def __repr__(self):
         """Represent the configured plugin instance as a string."""
-        result = ('<WetBulbTemperatureIntegral: precision: {}, '
-                  'coord_name_to_integrate: {}, start_point: {}, '
-                  'end_point: {}, direction_of_integration: {}>'.format(
-                      self.precision, self.coord_name_to_integrate,
-                      self.start_point, self.end_point,
-                      self.direction_of_integration))
+        result = ('<WetBulbTemperatureIntegral: precision: 0.005, '
+                  'coord_name_to_integrate: height, start_point: None, '
+                  'end_point: None, direction_of_integration: negative>')
```

Comment:
Surely these values shouldn't be hard-coded?

Reply:
True. I've corrected this.